EIPS/eip-4760.md

```diff
@@ -0,0 +1,54 @@
+---
+eip: 4760
+title: SELFDESTRUCT bomb
+description: Deactivate SELFDESTRUCT by changing it to SENDALL, which does recover all funds to the caller but does not delete any contract code or storage. In order to give applications good time to switch to new constructions, do this via a stage of exponential gas cost increases.
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin), Dankrad Feist (@dankrad)
+discussions-to: https://github.com/ethereum/EIPs/issues/4761
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-03
+requires: 
+---
+# SELFDESTRUCT bomb
+
+## Summary
+
+This EIP renames the `SELFDESCRUCT` opcode to `SENDALL`, and replaces its functionality. The new functionality will be only to send all Ether in the account to the caller.
+
+In order to give apps more warning even if their developers are completely unaware of the EIP process, this version will exponentially increase the gas costs of the opcode, so any developer has time to see this change and react by implementing a version of their contract that does not rely on `SELFDESTRUCT` .
+
+## Motivation
+
+The `SELFDESTRUCT` opcode requires large changes to the state of an account, in particular removing all code and storage. This will not be possible in the future with Verkle trees: Each account will be stored in many different account keys, which will not be obviously connected to the root account (see [Verkle tree EIP](https://notes.ethereum.org/5HDhQXstTaKtVqVbS7S9yw)).
```

Comment:
Alternatively, finish creating an actual Draft EIP (not just a HackMD) for the Verkle tree spec and add it the Requires list.